src/scss/pages/_courses.scss

```diff
@@ -0,0 +1,459 @@
+@use 'base/breakpoints' as *;
+@use 'base/functions' as *;
+
+// main (landmark)
+
+main {
```

Comment:
This will set the rule to all `main` elements across the site, not just the courses index...

I think it's fine for now, but maybe add a TODO to move it to a more "global" place.